src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/aktivitet/RegisterAktivitetController.kt

```diff
@@ -15,18 +15,18 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping(path = ["/api/aktivitet"])
```

Comment:
Burde kanskje endre path her? 
```suggestion
@RequestMapping(path = ["/api/aktivitet", "/api/register-aktivitet"])
```

Reply:
Ahaa smart